### PR TITLE
feat(SearchInput): Allow passing custom form elements after the attributes

### PR DESCRIPTION
**Pull request by KKoukiou**
Can be used to link to docs pages for example.

This is for cockpit, we want to use it like this:
Screen Shot 2021-07-16 at 10 17 08

See the last item, it's basically the docs link and and CopyClipboard with the CLI command equivalent generated by the user's input.

### CODE REVIEW

**Review by redallen — DISMISSED** [review]

File: packages/react-core/src/components/SearchInput/SearchInput.tsx [hunk 1]
```diff
@@ -29,6 +29,8 @@ export interface SearchInputProps extends Omit<React.HTMLProps<HTMLDivElement>,
   value?: string;
   /** Array of attribute values */
   attributes?: string[] | SearchAttribute[];
+  /* Additional elements added after the attributes in the form */
```
Comment (nicolethoen) [inline packages/react-core/src/components/SearchInput/SearchInput.tsx:32]: Could you add the suggestion to wrap additional form elements in a FormGroup component for formatting?

**Review by nicolethoen — APPROVED** [review]
We may also just want to update the advanced form unit test to include testing this new prop.

**Review by nicolethoen — COMMENTED** [review]
Sorry, I meant to leave a comment, not approve quite yet. Wrong button.

File: packages/react-core/src/components/SearchInput/SearchInput.tsx (hunk 1 above)
Comment (KKoukiou) [reply packages/react-core/src/components/SearchInput/SearchInput.tsx]: Fixed, let me know if you have a better wording suggestion :)

Comment (nicolethoen) [reply packages/react-core/src/components/SearchInput/SearchInput.tsx]: Looks good. Could you also update the advanced form unit test to include testing this new prop?

Comment (KKoukiou) [reply packages/react-core/src/components/SearchInput/SearchInput.tsx]: definitely, forgot about the tests, sorry !

**Review by redallen — APPROVED** [review]

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**